custom_components/tuya_local/devices/eurom_sani_bathroom_towel_radiator.yaml

```diff
@@ -108,5 +108,9 @@ secondary_entities:
     category: config
     dps:
       - id: 102
-        name: value
         type: boolean
+        mapping:
```

Comment:
This is incorrect, the value should be a boolean, not a string.

The correct fix would be to change the "name: value", which you removed, to "name: switch"